api/tests/opentrons/protocol_engine/clients/test_sync_client.py

```diff
@@ -295,15 +295,23 @@ def test_drop_tip(
     """It should execute a drop up tip command."""
     request = commands.DropTipCreate(
         params=commands.DropTipParams(
-            pipetteId="123", labwareId="456", wellName="A2", wellLocation=WellLocation()
+            pipetteId="123",
+            labwareId="456",
+            wellName="A2",
+            wellLocation=WellLocation(),
+            homeAfter=None,
```

Comment:
Since `None` is the default, let's use `True` to make sure `SyncClient` doesn't forget to set `homeAfter`.

```suggestion
            pipetteId="123",
            labwareId="456",
            wellName="A2",
            wellLocation=WellLocation(),
            homeAfter=True,
```